ste/xfer-anyToRemote-file.go

```diff
@@ -589,6 +590,8 @@ func commonSenderCompletion(jptm IJobPartTransferMgr, s sender, info TransferInf
 		// and we know the transfer didn't fail (because just checked its status above and made sure the context was not canceled),
 		// so it must have succeeded. So make sure its not left "in progress" state
 		jptm.SetStatus(common.ETransferStatus.Success())
+		jptm.CheckPoint().TransferDone() // Better place for this line would be Epilogue. But because we've not yet set the
```

Comment:
In between if context is cancelled or user aborts the job, we need to clean all this bitmap and other house keeping stuff.